src/commands/RunCommand.js

```diff
@@ -42,8 +42,7 @@ export default class RunCommand extends Command {
   }
 
   runScriptInPackage(pkg, callback) {
-    NpmUtilities.runScriptInDir(this.script, this.args, pkg.location, (err, stdout) => {
-      this.logger.info(stdout);
+    NpmUtilities.runScriptInDir(this.script, this.args, pkg.location, (err) => {
       if (err) {
```

Comment:
in order to get `stdout` on `bootstrap` errors, we'd have to modify how each `callback` or `cb` is called to include a second parameter (passed from `exec`). i can't think of a scenario in which always logging `stdout` on errors will cause an issue.